core/sup_tree_core/periodic_log_writer.ex

```diff
@@ -0,0 +1,150 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
```

Comment:
It is a bit confusing to me that `core/sup_tree_core/periodic_log_writer.ex` contains three modules (and each module is not so small).
How about rearranging files and modules?
For example:
```
sup_tree_core/
└── periodic_log/
    ├── message_builder.ex
    ├── reduction_builder.ex
    └── writer.ex
```

Reply:
Thanks.

Reply:
Fixed 15d399e0d4d85fba0e6f2a082bc8a90f495f6999